srcpkgs/synapse/template

```diff
@@ -14,15 +15,19 @@ depends="python3-jsonschema python3-frozendict python3-canonicaljson
  python3-setuptools python3-bleach python3-Jinja2 python3-psycopg2 python3-lxml
  python3-saml2 python3-treq python3-macaroons python3-sortedcontainers
  python3-typing_extensions python3-ijson python3-matrix-common
- python3-packaging python3-pydantic"
+ python3-packaging python3-pydantic python3-setuptools-rust"
```

Comment:
Why is `python3-setuptools-rust` a run-time dependency?

Reply:
What a mess; they can't properly specify build-time dependencies and work around the problem by abusing run-time dependencies.

Please try patching out the apparently needless setuptools_rust dependency in `pyproject.toml` and confirm that the program runs as expected. We don't need to perpetuate failures of the Python packaging ecosystem.